proposals/0304-structured-concurrency.md

```diff
@@ -609,17 +606,17 @@ A new, detached task can be created with the `detach` operation. The resulting t
 ```swift
 /// Create a new, detached task that produces a value of type `T`.
 @discardableResult
-static func detach<T: Sendable>(
+public func detach<T>(
   priority: Task.Priority = .unspecified,
-  operation: @escaping @concurrent () async -> T
+  operation: @Sendable @escaping () async -> T
 ) -> Task.Handle<T, Never>
 
 /// Create a new, detached task that produces a value of type `T` or throws an error.
 @discardableResult
-static func detach<T: Sendable>(
```

Comment:
thanks for removing the `static`, we don't usually do `public` for everything in proposals -- every single function would have it. I would prefer not adding them, thanks